lib/intercom/intercom.go

```diff
@@ -44,3 +44,14 @@ func (svc *Client) SendEvent(userID, eventName string, metadata map[string]inter
 		Metadata:  metadata,
 	})
 }
+
+// SaveUser accepts a UserID and CustomAttributes and forwards the request to Intercom
+// Intercom will create or update the user
+func (svc *Client) SaveUser(userID string, customAttributes map[string]interface{}) error {
```

Comment:
Is this a cuvva user ID or an intercom user ID?

Reply:
cuvva user id. feature works, tested it on avo.